From 7819f50859749aa0e501c496a5e371711c475b76 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 07 Aug 2018 11:19:56 +0300
Subject: [PATCH] board: habana: remote_run: use one tee command for case

Lower the amount of 'tee' commands by using one for the whole case.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/remote_run b/board/habanalabs/common/remote_run
index 075230c..313a57a 100755
--- a/board/habanalabs/common/remote_run
+++ b/board/habanalabs/common/remote_run
@@ -55,18 +55,18 @@
 	local pipestat=${PIPESTATUS[0]}
 	case $pipestat in
 	0)
-		print_msg "${scr_name[0]}: returned success" | tee -a $result
+		print_msg "${scr_name[0]}: returned success"
 		;;
 	1)
-		print_msg "${scr_name[0]}: Failed installing packages" | tee -a $result
+		print_msg "${scr_name[0]}: Failed installing packages"
 		;;
 	2)
-		print_msg "${scr_name[0]}: Failed programming SPI flash" | tee -a $result
+		print_msg "${scr_name[0]}: Failed programming SPI flash"
 		;;
 	*)
-		print_msg "${scr_name[0]}: Unknown error" | tee -a $result
+		print_msg "${scr_name[0]}: Unknown error"
 		;;
-	esac
+	esac | tee -a $result
 
 	print_msg "${scr_name[0]} script status: $pipestat" | tee -a $result
 	return $pipestat;
